Makefile

```diff
@@ -127,41 +127,22 @@ prepare-transformers:
 	CACHE_DIR=$$HOME_DIR/.cache/torch/transformers;\
 	mkdir -p "$$CACHE_DIR";\
 	i=0;\
-	while read -r URL; do read -r CACHE_FILE; if { [ $(CI) ]  &&  [ $$i -gt 4 ]; } || ! [ $(CI) ]; then wget $$URL -O $$CACHE_DIR/$$CACHE_FILE; fi; i=$$((i + 1)); done < "data/test/hf_transformers_models.txt"
-
-prepare-tests-files: prepare-spacy prepare-mitie install-mitie prepare-transformers
-
-prepare-wget-macos:
-	brew install wget || true
+	while read -r URL; do read -r CACHE_FILE; if { [ $(CI) ]  &&  [ $$i -gt 4 ]; } || ! [ $(CI) ]; then wget -nv $$URL -O $$CACHE_DIR/$$CACHE_FILE; fi; i=$$((i + 1)); done < "data/test/hf_transformers_models.txt"
 
-prepare-tests-macos: prepare-wget-macos prepare-tests-files
-	brew install graphviz || true
+prepare-tests-macos:
+	brew install wget graphviz || true
 
-# runs install-full target again in CI job runs, because poetry introduced a change
-# in behaviour in versions >= 1.2 (whenever you install a specific extra only, e.g.
-# spacy, poetry will uninstall all other extras from the environment)
-# See discussion thread: https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1667924056444669
```

Comment:
for context: `install-full` is not necessary to run again bc we don't deinstall things anymore when downloading spacy models